lang/java.md

```diff
@@ -36,66 +38,67 @@ everywhere!
 
 - Used for Server-Side Technologies like [Apache](https://commons.apache.org/), [JBoss](https://developers.redhat.com/products/eap/overview), [GlassFish](https://javaee.github.io/glassfish/), etc.
     
-## Why to use Java?
-Java was designed to be easy to use and is therefore easy to write, compile, debug, and learn than other programming languages. Java is object-oriented. This allows you to 
-create modular programs and reusable code. Java is platform-independent.
+## 🤔 Why to use Java?
```

Comment:
```suggestion
## 🤔 Why use Java?
```

Reply:
@Panquesito7: This can't be modified as mentioned in [README.md](https://github.com/devstrons/hello-world/blob/main/README.md).

Reply:
Hm? What do you mean? `Why to use Java?` has incorrect grammar.

Reply:
@Panquesito7: As per your suggestion `Why use Java?`, is bad to read!

**Note:** The grammar is good for both the use cases, we have to write it in a form where folks/beginners feel more friendly/comfortable, that's not sound like bad grammar, & it's doing!

Reply:
I agree with @amino19 that `Why to use Java?` is proper grammar and sounds better. `Why use Java?` feel incomplete to me 😅